src/terminal/dune

```diff
@@ -1,4 +1,9 @@
 (library
- (name terminal)
  (public_name terminal)
- (libraries stdlib-shims uutf uucp))
+ (name terminal)
+ (foreign_stubs
+  (language c)
+  (names terminal_stubs))
+ (libraries
+  (re_export terminal_ansi)
```

Comment:
I'm not sure to really understand the `re_export` field. That means that if `unix` is found as a dependency, we integrate `terminate_ansi`?

Reply:
I don't understand `re_export` either and copied that trick from `progress/dune`, but it doesn't work without it as e.g. `Terminal.Color` is now a module alias to `Terminal_ansi.Color` and without `re_export` the `Terminal_ansi` would be missing and cause compilation error (so a user would need to always depend on both `terminal terminal.ansi` rather than just `terminal`)

(The `re_export` doesn't have anything to do with `unix` being available, a mirageos unikernel would depend on `terminal.ansi` and ignore the unix-dependent `terminal` lib, just like it would need to depend on `progress.engine` and wouldn't be able to use the unixy `progress` lib)